mita/async.py

```diff
@@ -61,8 +61,17 @@ def check_idling():
             uuid = n['name'].split('__')[-1]
             if node_info.get('idle'):
                 logging.info("found an idle node: %s" % n['name'])
-                node_endpoint = get_mita_api('nodes', uuid, 'idle')
-                requests.post(node_endpoint)
+                # check to see if this node is still in the db
+                node = models.Node.query.filter_by(identifier=uuid).first()
+                if node:
+                    # node is still in the db, mark it as idle
+                    node_endpoint = get_mita_api('nodes', uuid, 'idle')
+                    requests.post(node_endpoint)
+                else:
```

Comment:
this `else` condition would delete any node added to jenkins that has a node with a name that contains `__` would get deleted.